Libraries/Utilities/deprecatedCallback.js

```diff
@@ -0,0 +1,54 @@
+/**
+ * Copyright (c) 2015-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ *
+ * @providesModule deprecatedCallback
+ * @flow
+ */
+
+'use strict';
+
+// @deprecated, will be removed on April 1 2016
```

Comment:
Yea, I'd just rather not commit us to a particular time frame on the utility function.  We may need something like this later for other things.